director/accounts/models.py

```diff
@@ -8,11 +13,25 @@
 
 
 class AccountPermissionType(EnumChoice):
+    """
+    Modification allows to link a new project to the account,
+    to create, update and delete teams related to the account.
+
+    Administration is a higher level of permission type. It allows to
+    delete the account; update the billing details for the account and
+    give other account members administrative perimission.
+    """
+
     MODIFY = 'modify'
     ADMINISTER = 'administer'
 
 
 class Account(models.Model):
+    """
+    Accounts are the entity against which resource usage is metered, and
+    can be billed if it exceeds the free allocation. Every user has their
+    own Account. 
+    """
     name = models.TextField()
```

Comment:
@apawlik : it would be good if you could work on the model fields too, adding `help_text` (which appears in admin and on view forms) and other parameters where appropriate. e.g.

```py
name = models.TextField(
        null=True,
        blank=True,
        help_text="The name of the account"
)
```

Reply:
Yip! Understood. Thanks for the feedback